src/OrbitQt/MultipleOfSpinBoxTest.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include "MultipleOfSpinBox.h"
+
+namespace orbit_qt {
+TEST(MultipleOfSpinBox, Validate) {
```

Comment:
This test should test the proposed subclass of `QIntValidator` and not the widget itself.

Reply:
Done.